.gitignore

```diff
@@ -51,3 +51,4 @@ src/nrnoc/syn.c
 src/nrnoc/vclmp.c
 src/oc/parse.cpp
 src/oc/parse.hpp
+share/lib/python/neuron/help_data.dat
```

Comment:
I wonder if it is better to think about getting rid of these instead of adding to them. What is the downside of just creating it in ```${PROJECT_BINARY_DIR}/lib/python/neuron/help_data.dat``` and simplifying the add_custom_target  below.

Reply:
That would be an elegant solution, but it would involve more work since current setup(CMake, setuptools, autotools) is relying on that path. I've noticed we have several similar handlings in our codebase (i.e. `hh.mod` which drives me crazy sometimes)